### PR TITLE
{Test} Bump azure-python-devtools to 1.2.0

### PR DESCRIPTION
**Pull request by haroldrandom**
**Description**  

Revert https://github.com/Azure/azure-cli/pull/13680 and ready to welcome the new version of azure-python-devtools.

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: .azure-pipelines/templates/automation_test.yml [hunk 1]
```diff
@@ -13,6 +13,8 @@ steps:
 
       source env/bin/activate
 
+      pip install git+https://github.com/haroldrandom/azure-python-devtools.git@fix-python-3.8.3
```
Comment (haroldrandom) [inline .azure-pipelines/templates/automation_test.yml:16]: Will revert it once https://github.com/Azure/azure-python-devtools/pull/65 is merged

**Comment by yonzhan** [comment]
Test

File: scripts/ci/test_automation.sh [hunk 2]
```diff
@@ -54,4 +54,7 @@ az -v
 az -h
 
 title 'Running tests'
+
+pip install git+https://github.com/haroldrandom/azure-python-devtools.git@fix-python-3.8.3
```
Comment (haroldrandom) [inline scripts/ci/test_automation.sh:58]: Will revert it once https://github.com/Azure/azure-python-devtools/pull/65 is merged

**Comment by bim-msft** [comment]
Version in the title should be 1.2.0?

**Comment by haroldrandom** [comment]
> Version in the title should be 1.2.0?

Fixed

**Pull request merged**